OracleIdentityGovernance/dockerfiles/12.2.1.4.0/container-scripts/createDomainAndStart.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
+# # Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+# #
+# # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+# #
+# # Author: OIM Development (<raminder.deep.kaler@oracle.com>)
```

Comment:
I would strongly encourage you not to include your personal email address in publicly acccessible code. If you really want this line, I would recommend using an alias or mailing list instead.

Reply:
taken care of